Engine/TransactionHandlers/BrokerageTransactionHandler.cs

```diff
@@ -629,6 +629,22 @@ private bool PerformCashSync()
                     return false;
                 }
 
+                // check if the brokerage is able to perform cash sync at this time
+                try
+                {
+                    if (!_brokerage.CanPerformCashSynchronization)
+                    {
+                        Log.Trace("BrokerageTransactionHandler.PerformCashSync(): Brokerage cannot perform cash sync at this time");
+                        _syncedLiveBrokerageCashToday = true;
+                        _lastSyncTimeTicks = CurrentTimeUtc.Ticks;
```

Comment:
This will cause a full day delay on cash sync and the equity curve will spike unreliably. If its repeatably being called at the same time each day (and its always closed) then it will actually never sync. 

Should rethink this and design a more robust "international" solution.

Reply:
Perhaps we need a map of IB-server locations to server reset times and we can move the IsWithinResetTimes() logic to be based on the server location.

Reply:
Why do you think it should delay/spike?
Cash sync is currently scheduled to be at `7:45 AM NY` every day, the only times `CanPerformCashSynchronization` would return `false` would be on Saturday and Sunday mornings (when cash sync cannot be performed anyway).

Reply:
RE: _rethink_ -- I know this is currently only an IB-centric issue, but it does seem as though the peculiarities of the brokerage implementations are leaking into the transaction handler. Perhaps more generally would be to have brokers perform their own cash syncs (via raising events) -- or maybe a much smaller and less risky change, the brokerage implementation could tell the transaction handler at what time the next sync is. The default implementation can just always return the next `7:45AM NY` -- but then internationally sensitive brokerages, such as IB, can return a time that properly observes the scheduled server outages.